ShellScript/Bash.sublime-syntax

```diff
@@ -558,7 +558,7 @@ contexts:
         2: keyword.operator.assignment.redirection.shell
 
   redirection-here-string:
-    - match: (\d*)(<<<)\s
+    - match: (\d*)(<<<)
```

Comment:
I believe we can still do something useful with e.g. a (negative) look-ahead here since shell operators (e.g. `<` or `&`) immediately following the `<<<` token are not allowed.

Reply:
The idea being that the match should not occur if there is a `<<<<` or `<<<&`?

Reply:
Exatly.

Reply:
The PR is ready to discuss. It superseeds all open PRs. So the question is just. Merge the open PRs, which are in a good shape first, or just close them in favor of the whole big patchset?

Reply:
@deathaxe It seems this is the only open ShellScript PR, so I'd say open a PR with your changes if they include the fix here and the extra check that @FichteFoll pointed out.

Reply:
Yes. I'll collect the issues to close and create a PR, hopefully soon.